pkg/controller/vsphere/util.go

```diff
@@ -25,10 +25,17 @@ const (
 // duplicating part of the type and parsing it ourselves using the same gcfg
 // library for now.
 type vSphereConfig struct {
-	Labels struct {
-		Zone   string `gcfg:"zone"`
-		Region string `gcfg:"region"`
-	}
+	Labels Labels `gcfg:"Labels"`
+	Global Global `gcfg:"Global"`
+}
+
+type Labels struct {
+	Zone   string `gcfg:"zone"`
+	Region string `gcfg:"region"`
+}
+
+type Global struct {
+	Port string `gcfg:"port"`
```

Comment:
I'm guessing port is better as a string, even though it will be a number in a string, might be worth a comment as to why it's a string?

Reply:
Port is a string because we need an empty string value here by default`fmt.Sprintf("%s:%s", providerSpec.Workspace.Server, getPortFromConfig(vSphereConfig))` and it seems like it's the cleanest way to do it